index.html

```diff
@@ -2370,13 +2370,40 @@ <h3>Unlinkability</h3>
       <section>
         <h3>Selective Disclosure</h3>
 
-        <p class="issue">
-When the contents of a digitally signed payload contains correlatable
-identifiers, those identifiers can be used to track individuals. A static
-digital signature is a correlatable identifier. There are selective
-disclosure digital signature schemes, such as BBS+, that are capable of
-not disclosing correlatable identifiers and ensuring that a different
-but valid digital signature is re-created upon every presentation.
+        <p>
+Selective disclosure is a technique that enables the recipient of a
+previously-signed message (that is, a message signed by its creator) to reveal
+parts of the message without revealing the entire message and without disturbing
+the verifiability of the origin of the parts being revealed. For example,
+selectively disclosing a digital driver's license for the purpose of renting a
+car might involve the entity presenting the license to only reveal the issuing
+authority, license number, birthday, and authorized motor vehicle class. Note
+that in this case, the license number is correlatable information that is being
+shared but some amount of privacy is still being preserved because the driver's
+full name and address are not being shared.
```

Comment:
Notice that some of the edits I suggested before didn't make it in. So, re-suggesting some of the edits I made before to make this less wordy:

```suggestion
only parts of the message without disturbing the verifiability of those
parts. For example, one might selectively disclose a digital driver's license for
the purpose of renting a car. This could involve revealing only the issuing
authority, license number, birthday, and authorized motor vehicle class from
the license. Note that in this case, the license number is correlatable
information, but some amount of privacy is preserved because the driver's
full name and address are not shared.
```